tests/_support/_command/ListCommands.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+namespace App\Commands;
```

Comment:
```suggestion
namespace Tests\Support\Command;
```

Reply:
No. It must be `App\Commands`.
Because it is for testing that `App\Commands\ListCommands` overrides the framework `list` command.